docs/using-hive.md

```diff
@@ -16,6 +16,32 @@
         $ oc create secret generic test-creds --from-literal=aws_secret_access_key=$AWS_SECRET_ACCESS_KEY --from-literal=aws_access_key_id=$AWS_ACCESS_KEY_ID
         ```
 
+### Using Global Pull Secret
+
+GlobalPullSecret can be used to specify a pull secret that will be used globally by all of the cluster deployments created by Hive. 
+When GlobalPullSecret is defined in Hive namespace and a cluster deployment specific pull secret (local pull secret) is sepcified, the contents of both pull secrets will be merged and then will be used for provisioning the OpenShift cluster. 
+In case same registry exists in both the pull secrets, precedence will be given to the contents of the local pull secret for the cluster deployment.
+
+Here are teh example commands to create a global pull secret and add it to Hive namespace
+
+```bash
+oc create secret generic global-pull-secret --from-file=.dockerconfigjson=<path>/config.json --type=kubernetes.io/dockerconfigjson --namespace hive
+```
+
+Following global secret information need to be added to the spec.
```

Comment:
"The global pull secret name must then be configured in the HiveConfig CRD:"

(move the oc edit command up before the spec yaml)